consensus/src/transaction_filter/mod.rs

```diff
@@ -106,7 +108,9 @@ mod test {
 
     fn get_module_name(txn: &SignedTransaction) -> String {
         match txn.payload().executable_ref() {
-            Ok(TransactionExecutableRef::EntryFunction(entry_func)) => {
+            Ok(TransactionExecutableRef::EntryFunction(entry_func))
+                if !txn.payload().is_multisig() =>
```

Comment:
we don't need to update this file since there's no filtering running

Reply:
Done.